frontend/src/pages/notebookController/NotebookController.tsx

```diff
@@ -28,16 +29,21 @@ export const NotebookController: React.FC = React.memo(() => {
     state.appState.user || '',
     state.appState.namespace || '',
   ]);
-  const projectName = ODH_NOTEBOOK_REPO || namespace;
+  const projectName = React.useMemo(
+    () => dashboardConfig.spec.notebookController?.notebookNamespace || namespace,
+    [namespace, dashboardConfig],
+  );
   const {
     notebook,
     loaded,
     loadError,
     forceUpdate: updateNotebook,
     setPollInterval: setNotebookPollInterval,
   } = useWatchNotebook(projectName);
-  const isNotebookRunning = !!(
-    notebook?.status?.readyReplicas && notebook?.status?.readyReplicas >= 1
+
+  const isNotebookRunning = React.useMemo(
+    () => !!(notebook?.status?.readyReplicas && notebook?.status?.readyReplicas >= 1),
+    [notebook],
```

Comment:
This is a fairly simple check -- and a primitive a that -- why do we want to memoize this?

Reply:
Already updated in #332